third_party/maya/lib/usdMaya/MayaMeshWriter.cpp

```diff
@@ -38,6 +38,49 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 PXRUSDMAYA_REGISTER_ADAPTOR_SCHEMA(MFn::kMesh, UsdGeomMesh);
 
+namespace {
+
+void _exportReferenceMesh(UsdGeomMesh& primSchema, MObject obj) {
```

Comment:
To match our style guidelines, could we place the return statements that are on single lines onto new lines, e.g.
```
if (!status) {
    return;
}
```